bindata/network/ovn-kubernetes/common/004-rbac-control-plane.yaml

```diff
@@ -61,6 +61,10 @@ rules:
   - egressips
   - egressservices
   - egressservices/status
+  - adminpolicybasedexternalroutes
+  - adminpolicybasedexternalroutes/status
+  - egressfirewalls
+  - egressfirewalls/status
```

Comment:
I guess we are following the status quo here but we should really only need update perms on `adminpolicybasedexternalroutes/status` and `egressfirewalls/status`